style/FeatherBB/view/footer.php

```diff
@@ -142,7 +142,7 @@
 
 </section>
 
-<script src="<?php echo get_base_url() ?>/style/FeatherBB/phone.min.js"></script>
+<script src="<?php echo get_base_url() ?>/js/phone.min.js"></script>
```

Comment:
Why did you change that? This file only needs to be loaded if we have FeatherBB style on